packages/core/components/ActionBar/ActionBar.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode, SyntheticEvent } from "react";
+import getClassNameFactory from "../../lib/get-class-name-factory";
+import styles from "./styles.module.css";
+const getClassName = getClassNameFactory("ActionBarComponent", styles);
+
+export const ActionBar = ({
+  label,
+  renderActions,
+  children,
+}: {
+  label?: string;
+  renderActions?: () => null;
+  children?: ReactNode;
+}) => (
+  <div className={getClassName()}>
+    {label && <div className={getClassName("actionsLabel")}>{label}</div>}
+    {children}
+  </div>
+);
+
+export const Action = ({
+  children,
+  onClick,
+}: {
+  children: ReactNode;
+  onClick: (e: SyntheticEvent) => void;
+}) => (
+  <button className={getClassName("action")} onClick={onClick}>
+    {children}
+  </button>
+);
+
+ActionBar.Action = Action;
```

Comment:
Not sure why this is showing up on the diff - suspect you might need a rebase

Reply:
Fixed.  It was an old file that didn't get removed